packages/nimbus/src/components/range-calendar/range-calendar.dev.mdx

```diff
@@ -0,0 +1,521 @@
+      <RangeCalendar
+        aria-label="Select date range"
+        isDateUnavailable={isWeekend}
+      />
+    </Stack>
+  );
+}
+```
+
+### Non-contiguous ranges
+
+Enable selection of ranges that span across unavailable dates with `allowsNonContiguousRanges`:
+
+```jsx-live-dev
+const App = () => {
+  // Mark Mondays as unavailable
+  const isMonday = (date) => {
+    return date.toDate(getLocalTimeZone()).getDay() === 1;
+  };
+
+  return (
+    <Stack direction="column" gap="400">
+      <Text textStyle="sm" color="neutral.11">
+        Mondays are unavailable, but ranges can span across them.
+      </Text>
+      <RangeCalendar
+        aria-label="Select date range"
+        allowsNonContiguousRanges={true}
+        isDateUnavailable={isMonday}
+      />
+    </Stack>
+  );
+}
+```
+
+### Custom week start
+
+Use `firstDayOfWeek` to change which day starts the week:
+
+```jsx-live-dev
+const App = () => (
+  <Stack direction="column" gap="400">
+    <Text textStyle="sm" color="neutral.11">
+      Week starts on Monday (common in European locales):
+    </Text>
+    <RangeCalendar
+      aria-label="Select date range"
+      firstDayOfWeek="mon"
+    />
+  </Stack>
+)
+```
+
+### Form states
+
+The component supports disabled and read-only states:
+
+```jsx-live-dev
+const App = () => {
+  const defaultRange = {
+    start: today(getLocalTimeZone()).add({ days: 3 }),
+    end: today(getLocalTimeZone()).add({ days: 8 }),
+  };
+
+  return (
+    <Stack direction="column" gap="600">
+      <Box>
+        <Text fontWeight="600" mb="200">Disabled:</Text>
+        <RangeCalendar
+          aria-label="Disabled calendar"
+          isDisabled
+          defaultValue={defaultRange}
+        />
+      </Box>
+      <Box>
+        <Text fontWeight="600" mb="200">Read-only:</Text>
+        <RangeCalendar
+          aria-label="Read-only calendar"
+          isReadOnly
+          defaultValue={defaultRange}
+        />
+      </Box>
+    </Stack>
+  );
+}
+```
+
+### Uncontrolled mode
+
+For simpler use cases, use uncontrolled mode with `defaultValue` and `onChange`:
+
+```jsx-live-dev
+const App = () => {
+  const [selectedRange, setSelectedRange] = useState<string>('No selection');
+
+  const formatDate = (date) => {
+    return date.toDate(getLocalTimeZone()).toLocaleDateString('en-US', {
+      month: 'short',
+      day: 'numeric',
+      year: 'numeric',
+    });
+  };
+
+  return (
+    <Stack direction="column" gap="400">
+      <RangeCalendar
+        aria-label="Select date range"
+        defaultValue={{
+          start: today(getLocalTimeZone()).add({ days: 3 }),
+          end: today(getLocalTimeZone()).add({ days: 8 }),
+        }}
+        onChange={(range) => {
+          if (range) {
+            setSelectedRange(`${formatDate(range.start)} to ${formatDate(range.end)}`);
+          } else {
+            setSelectedRange('No selection');
+          }
+        }}
+      />
+      <Text textStyle="sm">{selectedRange}</Text>
```

Comment:
It would be cool if the `selected range` was displayed the same in both examples

<img width="671" height="1083" alt="Image" src="https://github.com/user-attachments/assets/71ecd210-93d7-45bf-90d6-fdc00ea9bae7" />